src/tests/Microsoft.PowerFx.Core.Tests/TestRunnerTests/InternalSetup.cs

```diff
@@ -26,6 +26,12 @@ internal class InternalSetup
         /// </summary>
         internal bool DisableMemoryChecks { get; set; }
 
+        /// <summary>
+        /// By default, we run the result back through the serializer and deserialize again to se if we get the same result.
+        /// For some tests, for example reserved words, the deserialization will not work properly.
+        /// </summary>
+        internal bool SkipDeserializeComparison { get; set; }

```

Comment:
>SkipDeserializeComparison [](http://example.com/codeflow?start=22&length=25)

Do we need the new flag - can we just infer this if the test case has an error that we shouldn't try serializing?

Reply:
?

Reply:
Tracking #1409 to remove this flag.

Reply:
Flag removed and tests skipped for now.  I'll make a note in 1409, but the tests should be unskipped when that is done.